lib/seattleflu/id3c/cli/command/reportable_conditions.py

```diff
@@ -104,10 +123,11 @@ def webhook(suffix):
 
                 responses = {'ncov-reporting': send_slack_post_request(record, slack_webhooks['ncov-reporting'])}
 
-                # Also send Childcare specific results to the #ncov-reporting-childcare channel
-                if (record.collection_set_name in childcare['collection_sets']):
-                    responses['ncov-reporting-childcare'] = send_slack_post_request(
-                        record, slack_webhooks['ncov-reporting-childcare'])
+                # Also send study-specific results to their respective channels
+                for project in projects:
+                    if (record.collection_set_name in project['collection_sets']):
+                        responses[project['slack_channel_name']] = send_slack_post_request(
+                            record, project['slack_webhook'])
```

Comment:
If we never expect a collection set name to be used to send to more than one custom channel, then you could break here to save a little time.